### PR TITLE
Add kosmi.io Presence 

### PR DESCRIPTION
**Pull request by rndrmu**
Home Page Status:
image

When the user is in a room:
image

Browsing public rooms:
image

Browsing the Home page:
image

### CODE REVIEW

File: websites/K/Kosmi/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    "url": [
+      "app.kosmi.io",
+      "kosmi.io"
+    ],
+    "description": {
+      "en": "Online Hangouts Made Easy."
+    },
+    "service": "kosmi.io",
+    "version": "1.0",
```
Comment (SlowLife1661) [inline websites/K/Kosmi/dist/metadata.json:15]: 1.0.0

**Review by SlowLife1661 — CHANGES_REQUESTED** [review]

**Comment by SlowLife1661** [comment]
Also why use function instead of putting them all in event *just a comment* xd

**Comment by rndrmu** [comment]
> 
> 
> Also why use function instead of putting them all in event _just a comment_ xd
you're right actually, thats why i shouldnt do things late at night xd

**Comment by SlowLife1661** [comment]
Alright I will not go any further. And you might want to fix all deepscan errors.

File: websites/K/Kosmi/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "url": [
+      "app.kosmi.io",
+      "kosmi.io"
+    ],
+    "description": {
+      "en": "Online Hangouts Made Easy."
+    },
+    "service": "kosmi.io",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/v8b700V.png",
```
Comment (Bas950) [inline websites/K/Kosmi/dist/metadata.json:16]: logo must be at least 512x512

File: websites/K/Kosmi/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "thumbnail": "https://i.imgur.com/v8b700V.png",
+    "color": "#FF0000",
```
Comment (Bas950) [inline websites/K/Kosmi/dist/metadata.json:18]: Thumbnail is a promotional banner or a screenshot of the website homepage.

File: websites/K/Kosmi/presence.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                details = "Browsing public rooms...";
+            }
+                
+            else if (location.pathname.includes("/room/")) {
+                details = "In a room";
+            }
+                    
+            break;
+      default: break;
+    }
```
Comment (Bas950) [inline websites/K/Kosmi/presence.ts:30]: Currently a low quality presence, add some more statues or some interactive information (Like reading something of an element to display in status)

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by rndrmu** [comment]
should be good now :)

**Review by ririxidev — CHANGES_REQUESTED** [review]
jesus christ, make variables for these all things that you pass in smallImageText etc etc, this code is unreadable xDD

**Review by Bas950 — APPROVED** [review]

File: websites/K/Kosmi/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    "url": [
+      "app.kosmi.io",
+      "kosmi.io"
+    ],
+    "description": {
+      "en": "Online Hangouts Made Easy."
+    },
+    "service": "kosmi.io",
+    "version": "1.0.1",
```
Comment (Bas950) [inline websites/K/Kosmi/dist/metadata.json:15]: version 1.0.0 on release

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**